source/content/dns.md

```diff
@@ -174,39 +173,25 @@ Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (t
 
 Pantheon provides IPv6 addresses for you to assign `AAAA` records to. If your DNS service does not support `AAAA` you can simply omit these records for now, but consider asking your provider to add functionality, as the need for IPv6 will only increase in the future.
 
-Some DNS service providers require expanded AAAA records. You can retrieve the expanded version for each AAAA record by using the [IPv6 conversion tool](http://ipv6-literal.com/) and copying the "IPv6 longest" result.
-
+Some DNS service providers require expanded `AAAA` records. You can retrieve the expanded version for each AAAA record by using the [IPv6 conversion tool](http://ipv6-literal.com/) and copying the "IPv6 longest" result.
 
 ### What about my `MX` records for email?
 
-Pantheon does not provide email services. Make sure your DNS records include an `MX` reacord that points to a subdomain (like `mail`), which in turn has an `A` or `CNAME` record pointing it to your email provider.
+Pantheon does not provide email services. Make sure your DNS records include an `MX` record that points to a subdomain (like `mail`), which in turn has an `A` or `CNAME` record pointing it to your email provider.
 
 ### What is the difference between an `A` and `CNAME` record?
 
 An `A` record points a domain name to an IPv4 address. A `CNAME` record points a domain name to *another* domain, but *does not* redirect to it.
 
-Pantheon will provide both `A` and `CNAME` values, depending on the domains provided:
+Pantheon provides `A` and `AAAA` values:
 
 ![DNS Values provided by the Pantheon Site Dashboard](../images/dashboard/dns-values.png)
 
-If your DNS provider doesn't allow `CNAME` records, you can use `nslookup` to determine the IP address for the domain value, and then supply that as an `A` record:
-
-```bash
-$ nslookup live-anita-drupal.pantheonsite.io
-Server:         192.168.1.1
-Address:        192.168.1.1#53
-
-Non-authoritative answer:
-live-anita-drupal.pantheonsite.io       canonical name = fe1.edge.pantheon.io.
-Name:   fe1.edge.pantheon.io
-Address: 23.185.0.1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8001::1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8000::1
-```
+### Why does my domain say "Setup Recommended?"
+
+If the Platform detects a CNAME record pointed to Pantheon, the **Status** in **Domains / HTTPS** will show as <span class="glyphicons glyphicons-alert text-warning"></span> **Setup Recommended**. Click **Details** to find the value detected for the CNAME on the line with `Remove this detected record` in the status.
```

Comment:
They also might need to add the required A/AAAA records, so how about changing the last sentence to something like:

Click **Details** to find the required values for A and AAAA records to add, and the CNAME detected to remove. Login to your DNS provider to make the recommended changes.

Reply:
I think I've resolved this in the latest commit https://github.com/pantheon-systems/documentation/pull/5643/commits/101eaf55de3d691e18630af2735106125113c3be